src/main/java/org/jitsi/videobridge/xmpp/MediaStreamTrackFactory.java

```diff
@@ -181,11 +289,32 @@ else if (SourceGroupPacketExtension.SEMANTICS_FID
                     Long primarySSRC = fidEntry.getKey();
                     Long rtxSSRC = fidEntry.getValue();
 
-                    RTPEncodingDesc[] encodings = new RTPEncodingDesc[1];
+                    int numOfTemporal, numOfSpatial;
+                    if (false && ENABLE_SVC)
```

Comment:
No! `true  && ENABLE_SVC` is specific to VP9 SVC as it will make the bridge think that there are 6 layers (2 spatial * 3 temporal), but that is not true for plain VP8.